host-bin/enter-chroot

```diff
@@ -71,6 +73,7 @@ while getopts 'bc:k:ln:t:u:x' f; do
     n) NAME="$OPTARG";;
     t) TARGET="$OPTARG";;
     u) USERNAME="$OPTARG";;
+    X) TMPXMETHOD="$OPTARG";;
```

Comment:
Done, I had it my original file but it didn't make it to the online version somehow.